apps/schematic/api/pyproject.toml

```diff
@@ -7,21 +7,22 @@ readme = "README.md"
 packages = [{include = "schematic_api"}]
 
 [tool.poetry.dependencies]
-python = "3.11.0"
+python = ">=3.9.0,<3.11"
```

Comment:
Can you pin the version of Python for better reproducibility? Unlike a library that we would make and that we should expect to be used in (external) projects that use different versions of Python, I don't think we expect this app (REST API) to be run on multiple versions of Python. Hence, pinning the version will make the life of developers easier by ensuring that every developer use the same version of Python.

Reply:
Yeah that makes sense. I was trying to resolve dependencies issue on my end and I simply borrowed the toml file from schematic.